terraform/root_variables.tf

```diff
@@ -0,0 +1,31 @@
+variable "az_count" {
+  default = 2
+}
+
+variable "default_aws_region" {
+  default = "eu-west-2"
+}
+
+variable "dns_zone" {
+  description = "DNS zone name e.g. tdr-management.nationalarchives.gov.uk"
+  default     = "tdr-management.nationalarchives.gov.uk"
+}
+
+variable "domain_name" {
+  description = "fully qualified domain name for the service"
+  default     = "grafana.tdr-management.nationalarchives.gov.uk"
+}
+
+variable "function" {
+  description = "forms the second part of the bucket name, eg. upload"
+  default     = "grafana"
+}
+
```

Comment:
For the `function` and `project` variables, they both mention bucket names but they're in the root_variables here so it's not clear what they refer to. Maybe the function one could reference the alb. I don't think you need to mention a bucket at all for the `project` variable, just leave it at "abbreviation for the project" What do you think?

Reply:
Reworded